pkg/payload/precondition/clusterversion/upgradeable.go

```diff
@@ -91,7 +88,7 @@ func (pf *Upgradeable) Run(ctx context.Context, releaseContext precondition.Rele
 	// if no cluster overrides have been set
 	if currentMinor == desiredMinor {
 		klog.V(4).Infof("Precondition %q passed: minor from the current %s matches minor from the target %s (both %s).", pf.Name(), currentVersion, releaseContext.DesiredVersion, currentMinor)
-		if condition := ClusterVersionOverridesCondition(clusterVersion); condition != nil {
+		if condition := ClusterVersionOverridesCondition(cv); condition != nil {
```

Comment:
Shifting [this][1] thread over.  @jottofar, I think this is the only call-site that was consuming the `clusterVersion` argument I'm removing in this PR.  We added it in b72e843974 (#364).  But [your concerns][1] seem to be about `history`, and `ClusterVersionOverridesCondition` is just looking at `spec.overrides`.  Does that address your concerns?

[1]: https://github.com/openshift/cluster-version-operator/pull/708#discussion_r766168966

Reply:
NM, you're right, I'm confusing it with `currentVersion`. Not sure why I didn't originally use `cv` and instead added that param other than it being an oversight.